proxy/certs/certs.go

```diff
@@ -196,18 +196,22 @@ func (s *RemoteCertSource) Local(instance string) (tls.Certificate, error) {
 	createEphemeralRequest := sqladmin.SslCertsCreateEphemeralRequest{
 		PublicKey: pubKey,
 	}
+	var (
+		tok    *oauth2.Token
+		tokErr error
```

Comment:
nit: tokErr isn't used outside of `if s.EnableIAMLogin { ... }` - we should probably either use `err` here or scope it to that block